.github/workflows/lint.yml

```
@@ -29,7 +29,7 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v2.4.0
       - name: Install Node.js
-        uses: actions/setup-node@v2.5.1
+        uses: actions/setup-node@v3
```

This would preferably be (something like):

```suggestion
        uses: actions/setup-node@v3.0.0
```

(The same goes for all the other instances)